guide/creating_element/testing.md

```diff
@@ -144,6 +144,54 @@ end
 
 Once again, we are comparing the pipeline's output with our reference file to assert that everything works correctly.
 
+### Customizing pipeline module
+
+[`Membrane.Testing.Pipeline`](https://hexdocs.pm/membrane_core/Membrane.Testing.Pipeline.html) allows us to customize every callback of [`Membrane.Pipeline`](https://hexdocs.pm/membrane_core/Membrane.Pipeline.html). Let's create Pipeline that will answer to ping messages. To do this, we must define our custom module.
+
+```elixir
+defmodule MockPipeline do
+  use Membrane.Pipeline
+  
+  def handle_other({:ping, from}, state) do
+      send(from, :pong)
+      {:ok, state}
+  end
+  
+  def handle_other(_message, state),
+    do: {:ok, state}
+end
+```
+
+In order to use callbacks defined in `MockPipeline`, we have to specify it in `Options`.
+
+```elixir
+Pipeline.start_link(%Pipeline.Options{
+  elements: [
```

Comment:
Maybe it would be a better idea to replace `elements` with `...` since reader has already read that once?